python/.pre-commit-config.yaml

```
@@ -0,0 +1,15 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: check-added-large-files
+
+-   repo: https://github.com/psf/black
+    rev: 22.3.0
+    hooks:
+    -   id: black
```

```suggestion
    -   id: black
-   repo: https://github.com/pre-commit/mirrors-prettier
    rev: v2.6.2
    hooks:
    -   id: prettier
```

Takes care of JSON, YAML and Markdown files too.